server/src/main/java/org/apache/iotdb/db/service/thrift/impl/ClientRPCServiceImpl.java

```diff
@@ -314,7 +314,8 @@ public TSStatus setStorageGroup(long sessionId, String storageGroup) {
               SESSION_MANAGER.getSessionInfo(sessionId),
               "",
               PARTITION_FETCHER,
-              SCHEMA_FETCHER);
+              SCHEMA_FETCHER,
+              config.getQueryTimeoutThreshold());
```

Comment:
It's a little bit strange to add query timeout in write operation. So you can add a new method in Coordinator which doesn't need timeout parameter and in that method just use default query timeout as parameter to call another method with timeout parameter.

Reply:
ok,I have added a new method in Coordinator which doesn't need timeout parameter